client/src/App.js

```diff
@@ -6,7 +6,11 @@ import Project from './Project';
 class App extends Component {
   constructor(props) {
     super(props);
-    this.state = {projects: []};
+    this.state = {
+      new_projects: [],
+      in_progress_projects: [],
+      done_projects: []
```

Comment:
This also seems very hard-coded. What happens if we decide to add another status? Could we write the code so that all that requires is a change to the database?

Reply:
I agree, but again - that requires some bigger changes that I'm not ready to make yet. I need to be able to dynamically create the columns in order to make the change you're asking for relevant. Adding a new status should add a new column.